### ISSUE TITLE
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '/root/bwdata/docker/docker-compose.yml'

### ISSUE THREAD

**Issue by ozment** [bitwarden/server#527]
After a fresh installation of a DigitalOcean Bitwarden droplet, first login I'm receiving:

> ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '/root/bwdata/docker/docker-compose.yml'

Also, It never asked me for my installation ID for hosting that I was informed to register for because I'd be asked for input of it.

I'd be appreciative of any help, thanks!

**Comment by kspearrin** [comment]
I've noticed that sometimes you have to wait a minute or two after the droplet is created, before logging in, for it to finish initializing.

**Comment by thestepafter** [comment]
Was anyone ever able to fix this? I waited 30 minutes and still getting the same error as reported by @SlurpsBurgerton.

**Comment by neilsf-1** [comment]
I have a very similar problem.

I initially had an error telling me "Unable to validate installation id. Problem contacting Bitwarden server.", and I got dumped back to a shell prompt. Ever since this, I cannot force-reinstall Bitwarden, and keep getting the same error of "ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '/root/bwdata/docker/docker-compose.yml'".

I have tried building a 'docker-compose.yml' file but BW errors out with strong parsing problems. 

Re-running './bitwarden.sh install' yields the output "Looks like Bitwarden is already installed at /volume1/docker/bwdata", but with the still missing 'docker-compose.yml' file, BW will return to the startup ERROR. 

Any thoughts?

**Comment by thestepafter** [comment]
@neilsf-1 I actually solved this by manually typing the key from Bitwarden into my terminal instead of cutting and pasting it. Basically the installer continues even if the key isn't valid but the installer doesn't account for this and some files do not get installed. Once I typed the key in everything installed and worked as expected. (I even pasted the key into a text editor with no formatting before pasting into my terminal.)
